README.md

```diff
@@ -4,10 +4,41 @@ A 'package manager' for Glorious Eggroll's custom proton build.
 
 ## Installation
 
+There are currently a couple of ways to install `humble-lumpia`.
+
+**NOTE:** Installing `humble-lumpia` and its dependencies is not sufficient to 
+run the versions of `proton-ge-custom` that `humble-lumpia` will download. Be 
+sure to reference the [README](https://github.com/GloriousEggroll/proton-ge-custom/)
+in `proton-ge-custom` to determine what other dependencies your system requires.
+
+### AUR Package Installation
+
+If you are running **Arch Linux**, you can also download and install the 
+package maintained by [NINNiT](https://github.com/NINNiT) from the AUR. 
+If you are unfamiliar with installing packages from the AUR, you can 
+review the process [here](https://wiki.archlinux.org/title/Arch_User_Repository#Installing_and_upgrading_packages)
```

Comment:
I think a simple copy + pastable example that works for most people would be valuable here. Eg.

```
yay -S humble-lumpia-git
```

It should probably go above the 'If you are unfamiliar with installing packages from the AUR.......' section

Reply:
I've added an example as you suggested.